### PR TITLE
Pruning ASIC section of schema

### PR DESCRIPTION
**Pull request by aolofsson**
- Removing maxcap, maxslew, maxfanout (duplication, belongs in  sdc)
- Making aspectratio defvalue to 1.0 (only used for auto generation, special cases always come with floorplan).
- Changing ignore to dontuse for cell list (better name)
- Removing load cell list (belongs in sdc)
- Moving asic,stackup to option, stackup (consistent with current PDK setting)
- Removing redundant asic,pdk value (was already in option)
- Moving lambda to pdk (natural grouping)

### CODE REVIEW

File: siliconcompiler/schema/schema_cfg.py [hunk 1]
```diff
@@ -311,6 +311,19 @@ def schema_pdk(cfg, stackup='default'):
             optimization. Node examples include 180, 130, 90, 65, 45, 32, 22 14,
             10, 7, 5, 3.""")
 
+    scparam(cfg,['pdk', pdkname, 'lambda'],
```
Comment (gadfort) [inline siliconcompiler/schema/schema_cfg.py:314]: It feels like this is intended to be a design parameters, so should it be in `option,lambda` instead?

File: siliconcompiler/schema/schema_cfg.py [hunk 2]
```diff
@@ -2112,6 +2116,27 @@ def schema_option(cfg):
             may load multiple flows and libraries.
             """)
 
+    scparam(cfg, ['option','pdk'],
+            sctype='str',
+            scope='job',
+            shorthelp="PDK target",
+            switch="-option_pdk <str>",
+            example=["cli: -option_pdk freepdk45",
```
Comment (gadfort) [inline siliconcompiler/schema/schema_cfg.py:2124]: other options don't have option prefix, should be `-pdk`

File: siliconcompiler/schema/schema_cfg.py [hunk 3]
```diff
@@ -2112,6 +2116,27 @@ def schema_option(cfg):
...
+                     "api: chip.set('option','pdk','freepdk45')"],
+            schelp="""
+            Target PDK used during compilation.""")
+
+    scparam(cfg, ['option', 'stackup'],
+            sctype='str',
+            scope='job',
+            shorthelp="Stackup target",
+            switch="-option_stackup <str>",
+            example=["cli: -option_stackup 2MA4MB2MC",
```
Comment (gadfort) [inline siliconcompiler/schema/schema_cfg.py:2134]: same, should be `-stackup`

**Review by gadfort — CHANGES_REQUESTED** [review]
overall looks good, just need to correct the cli switches and discuss the location of lambda

File: siliconcompiler/schema/schema_cfg.py [hunk 4]
```diff
@@ -2034,6 +2035,9 @@ def schema_option(cfg):
     ''' Technology agnostic run time options
     '''
 
+
+
+
```
Comment (WRansohoff) [inline siliconcompiler/schema/schema_cfg.py:2040]: Nit: delete?

File: siliconcompiler/targets/asap7_demo.py [hunk 5]
```diff
@@ -27,25 +27,20 @@ def setup(chip):
     #3. Select default flow/PDK
     chip.set('option', 'flow', 'asicflow')
     chip.set('option', 'pdk', 'asap7')
+    chip.set('option', 'stackup', '10M')
```
Comment (WRansohoff) [inline siliconcompiler/targets/asap7_demo.py:30]: Is `10M` right for asap7, or is it possible we accidentally copied it from freepdk45?

**Review by WRansohoff — APPROVED** [review]
LGTM, just a couple of minor comments.

**Review by nmoroze — APPROVED** [review]

File: siliconcompiler/schema/schema_cfg.py (hunk 2 above)
Comment (aolofsson) [reply siliconcompiler/schema/schema_cfg.py]: NIce catch, fixed.

File: siliconcompiler/schema/schema_cfg.py (hunk 3 above)
Comment (aolofsson) [reply siliconcompiler/schema/schema_cfg.py]: Fixed

**Review by gadfort — APPROVED** [review]

**Pull request merged**